README.md

```diff
@@ -1,5 +1,6 @@
 # Gradient - Politechnika Gdańska
-Witaj w repozytorium strony internetowej koła naukowego "Gradient". Poniżej zamieściliśmy kilka informacji, które mogą się przydać podczas Twojej kontrybucji. Prosimy Cię o dostosowanie się do panującego regulaminu!
+Witaj w repozytorium strony internetowej koła naukowego "Gradient". Poniżej zamieściliśmy kilka informacji,
+ które mogą się przydać podczas Twojej kontrybucji. Prosimy Cię o dostosowanie się do panującego regulaminu!
```

Comment:
Kontrybucja to takie spolczanie na siłę. Generalnie oznacza to zapłatę za przegraną wojnę :P Można to zastąpić jakimś "wkładem w rozwój" czy czymś innym o bardziej jasnym wyrazie.

Reply:
Przypomina mi to rozprawę "blacklist vs. denylist"😂
https://github.com/facebook/react/issues/13604